REST_API_Docs.md

```
@@ -0,0 +1,1441 @@
+		"message": "닉네임 형식이 올바르지 않습니다."
+	}
+}
+```
+
+**Response (409 Conflict)**
+
+```json
+{
+	"status": "error",
+	"error": {
+		"code": "DUFLICATE_NICKNAME",
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)